### DISCUSSION TITLE
Backing up and restore directus configurations only?

### DISCUSSION THREAD

**Discussion by n-pn** [directus/directus#4363] [Feature Requests]
I want to sync all the changes make in local directus app to the production server, is there any easy way to do it?

Backing up the database and restore from the sql is good enough, but I only want directus metadata, not the random collections data for local testing.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
